dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/CustomFormParam.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useMemo } from "react";
+import { IBasicFormParam } from "shared/types";
+import { CustomComponent } from "../../../RemoteComponent";
+
+export interface ICustomParamProps {
+  param: IBasicFormParam;
+  handleBasicFormParamChange: (
+    p: IBasicFormParam,
+  ) => (e: React.FormEvent<HTMLInputElement>) => void;
+}
+
+export default function CustomFormComponentLoader({
+  param,
+  handleBasicFormParamChange,
+}: ICustomParamProps) {
+  // Fetches the custom-component bundle served by the dashboard nginx
+  const url = `${window.location.origin}/custom_components.js`;
```

Comment:
@andresmgot @aanthonyrizzo this request happens in the browser, correct? Doesn't this file need to be versioned/fingerprinted in order to bust browser cache when custom components bundle contents/file changes?

Reply:
I've not explicitly tried, but I wouldn't think so. The loading is using https://github.com/Paciolan/remote-component#src/RemoteComponent.js which uses remote-module-loader which uses fetch (by default) with the default cache option. If you find it is, we can update the config for the loader it seems.